Sources/WordPressKit/Models/RemotePostParameters.swift

```diff
@@ -198,10 +198,14 @@ private enum RemotePostWordPressComCodingKeys: String, CodingKey {
     static let postTags = "post_tag"
 }
 
-struct RemotePostCreateParametersWordPressComEncoder: Encodable {
+public struct RemotePostCreateParametersWordPressComEncoder: Encodable {
     let parameters: RemotePostCreateParameters
 
-    func encode(to encoder: Encoder) throws {
+    public init(parameters: RemotePostCreateParameters) {
+        self.parameters = parameters
+    }
+
+    public func encode(to encoder: Encoder) throws {
```

Comment:
When the type became `public`, a `public init` became necessary because the compiler does not synthesize one outside of the package.

As for `encode(to:)`, it needs to be `public` because `Encodable` is declared as part of the `public` type interface. 

🤔 In hindsight, I suppose the `Encodable` part could be moved into an `internal` extension, but I think it's okay to leave it as is. Let me know if you can think of a strong reason to hide it.